### ISSUE TITLE
Add the ability to add a speed camera or a surveillance camera

### ISSUE THREAD

**Issue by Growebis** [organicmaps/organicmaps#9383]
⚠ Have you searched for similar, already existing issues?

**Is your feature request related to a problem? Please describe.**

Sometimes i notice a speed camera or a normal camera on the road but often times i have to favorite my location and then write down what the camera is and then go on OSM and add it which is quite annoying and slow and clutters up my map with a bunch of favorites.

**Describe the ideal solution**

I want to be able to add a camera and then in it there would be some options to be able to change what kind of camera it is and what it's used for.

**Describe alternatives you have considered**
none

**Additional context**

**Comment by therealrobster** [comment]
I agree, a little like Waze, where you can report cameras.  I mean, if you could have user agreement, like in Waze, that would be a game changer.  Probably a LOT of work and resources though?
